css/styles.css

```diff
@@ -0,0 +1,4 @@
+* {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
- [ ] As provided in the project requirements, kindly give your h1 element in your index.html have a class and use a class selector to style it.
![123](https://user-images.githubusercontent.com/93598189/191290685-dcb9f8ea-294a-4bcf-b00d-9b816786cfa6.png)